apps/dav/tests/unit/CalDAV/CalDavBackendTest.php

```diff
@@ -402,6 +402,7 @@ public function providesCalendarQueryParameters() {
 			'all' => [[0, 1, 2, 3], [], []],
 			'only-todos' => [[], ['name' => 'VTODO'], []],
 			'only-events' => [[0, 1, 2, 3], [], [['name' => 'VEVENT', 'is-not-defined' => false, 'comp-filters' => [], 'time-range' => ['start' => null, 'end' => null], 'prop-filters' => []]],],
+			'timernage false' => [[0, 1, 2, 3], [], [['name' => 'VEVENT', 'is-not-defined' => false, 'comp-filters' => [], 'time-range' => false, 'prop-filters' => []]],],
```

Comment:
```suggestion
			'timerange false' => [[0, 1, 2, 3], [], [['name' => 'VEVENT', 'is-not-defined' => false, 'comp-filters' => [], 'time-range' => false, 'prop-filters' => []]],],
```

Typo in the key - actually it makes no difference to the test data and result.